homework.txt

```diff
@@ -0,0 +1,26 @@
+problem 1
+function calculator(num1, num2, operation){
+var num1 = 1;
+var num2 = 2;
+var operation ;
```

Comment:
Good work @Salem8809 .You don't need to redefine `num1, num2, or operation`. They've already been passed to the function as arguments, which means that your function has access to them. Your code can be:
```js
function calculator(num1, num2, operation){
    if (operation === "add" ){
return num1 + num2;
    } if (operation === "subtract"){
return num1 - num2;
    } if (operation === "multiply"){
return num1 * num2;
    }if  (operation === "divide"){
return num1 / num2;
    }
}

```